### DISCUSSION TITLE
Favorites e621

### DISCUSSION THREAD

**Discussion by arkuid** [Official-Husko/NN-Downloader#6] [Q&A] [answered]
Hello, how do I download favorites from the site e621? The program only waits for tags.

**Comment by Official-Husko** [comment, marked as answer]
Make sure to have your API Key configured properly then use "fav:USERNAME".

**Comment by arkuid** [comment]
Thanks. It's my fault that I didn't get acquainted with the site tags. (../wiki_pages/9169)

**Reply by Official-Husko** [reply]
All good, I'm glad to hear it worked out for you. Enjoy the tool :)

**Comment by Kubboy** [comment]
Hey trying to download form e621 but I don't know how to find my API can you help me?

**Reply by Official-Husko** [reply]
Sure thing, to use the API you will need an account so make sure you got that and that you are logged into it. After that you can click on "Account > Manage API Access" and in there you can generate an API Key.

If you need further help feel free to comment on what part you are stuck on.
